### ISSUE TITLE
Advice about incompatibility after update

### ISSUE THREAD

**Issue by yige-yigeren** [valinet/ExplorerPatcher#1498]
run gpedit.msc (This feature is only available in the professional version. but you can go [google](https://www.google.com/search?q=how+to+enable+gpedit.msc+in+home+verson) how to enable it on home edition)
Computer Configuration - Administrative Templates - Windows Components - Windows Update - Manage updates available from Windows Update
Configure quality and feature updates to be deferred for 14 days.
This basically avoids incompatibility.
At the same time, you can guarantee that you can receive updates normally - just delayed a little.
Under no circumstances do I recommend turning off updates completely. If a feature you don't like is updated, there is a way to roll it back instead of turning off updates completely.
Maybe write this into the readme? This can reduce a lot of issues and crashes.

**Comment by Heeere-GittyGitty** [comment]
You're totally right, I've crashed two machines after the latest cumulative update on Windows.
I'll open an Issue on this here, and explain the workaround i've found

**Comment by Heeere-GittyGitty** [comment]
Now with detail here:
https://github.com/valinet/ExplorerPatcher/issues/1499

**Comment by pyrates999** [comment]
I recommend you do not do this because updates are released for a reason.  Especially if there is a fix for a zero day exploit.

**Comment by yige-yigeren** [comment]
> I recommend you do not do this because updates are released for a reason. Especially if there is a fix for a zero day exploit.

I'm not very clear about the various parts of the update, but as far as I know, there are different parts of the update, security updates can be automatically installed by many antivirus programs, and with an appropriate delay (about 1~2 weeks), there will be no at this time Excessive security exploits. I think it is necessary to sacrifice the security brought by a little update to prevent your explorer from crashing.

I don't know whether this program can add the function of stopping injection and modification after the version is found to be unsupported, but we can only slightly delay our own system update to avoid unsuitability.

**Comment by yige-yigeren** [comment]
> 我建议您不要这样做，因为发布更新是有原因的。特别是如果有针对零日漏洞的修复程序。

And many times Microsoft's automatic update will download some problematic updates. A delay of 1~2 weeks will help you know whether there will be problems with these updates from various news channels before downloading these updates. IMHO, Windows has multiple updates Nearly ruined my computer, including deleting some frameworks causing program exceptions, OneDrive crashed completely, file system crashed (normal characters turned into various characters, causing me to be unable to create/move files)

**Comment by pyrates999** [comment]
It sounds like you need to do a reinstall of windows if updates are causing this many problems.

**Comment by yige-yigeren** [comment]
> 听起来如果更新导致这么多问题，您需要重新安装 Windows。

In fact, I have reinstalled the system twice due to the system abnormality caused by the update, so I will always install edgeless (a networked PE that can work temporarily) and images of various versions of windows in one of my USB flash drives.

@.@So actually I would defer windows updates by default, security updates would be automatically installed by antivirus software anyway.

**Comment by pyrates999** [comment]
Have you thought it might be a problem with your hardware then?  This many problems might point to:

- bad ram
- bad cpu
- bad motherboard
- bad gpu
- bad ssd
- bad psu

**Comment by yige-yigeren** [comment]
> Have you thought it might be a problem with your hardware then? This many problems might point to:
> 
> * bad ram
> * bad cpu
> * bad motherboard
> * bad gpu
> * bad ssd
> * bad psu

In fact, sometimes after Microsoft has updated explorer, ExplorerPatcher has not yet supported the new version, and it is likely to cause explorer to crash completely. Postponing the update is the most effective way.
